app/common/state/bookmarksState.js

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Immutable = require('immutable')
+
+// Constants
+const dragTypes = require('../../../js/constants/dragTypes')
+
+// Utils
+const bookmarkUtil = require('../lib/bookmarkUtil')
+const domUtil = require('../../renderer/lib/domUtil')
+const siteUtil = require('../../../js/state/siteUtil')
+const {calculateTextWidth} = require('../../../js/lib/textCalculator')
+const {iconSize} = require('../../../js/constants/config')
+
+const bookmarksState = {
+  getDNDBookmarkData: (state, bookmark) => {
```

Comment:
I think these probably make more sense in bookmarkUtil. Bookmarks don't have their own state right now because they are part of `sites`